compiler/luci/pass/include/luci/Pass/QuantizeWithMinMaxPass.h

```diff
@@ -45,9 +45,10 @@ class QuantizeWithMinMaxPass : public logo::Pass
 public:
   QuantizeWithMinMaxPass(loco::DataType input_model_dtype, loco::DataType output_model_dtype,
                          QuantizationGranularity granularity, loco::DataType input_type,
-                         loco::DataType output_type)
+                         loco::DataType output_type, bool TF_style_maxpool)
```

Comment:
How about (1) don't add `TF_style_maxpool` (2) set `_TF_style_maxpool` as `false` (3) provide attribute to change `_TF_style_maxpool` ?